telegraf/1.21/alpine/Dockerfile

```diff
@@ -1,20 +1,26 @@
 FROM alpine:3.15
 
 RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
-RUN apk add --no-cache iputils ca-certificates net-snmp-tools procps lm_sensors tzdata su-exec libcap && \
+RUN apk add --no-cache \
+      ca-certificates  \
+      gnupg            \
+      iputils          \
+      libcap           \
+      lm_sensors       \
+      net-snmp-tools   \
+      procps           \
+      su-exec          \
+      tzdata           && \
     update-ca-certificates
 
+RUN gpg --yes --batch \
+      --keyserver hkps://keyserver.ubuntu.com:443 \
+      --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5
+
 ENV TELEGRAF_VERSION 1.21.4
 
 RUN set -ex && \
-    mkdir ~/.gnupg; \
-    echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf; \
```

Comment:
I didn't but any reason to remove this? Or do we know why this was here in the first place?

Reply:
I probably need to look into this a bit more. The commit that initially added this claimed to work around a GPG issue, but I couldn't find what that issue was. It's possible it's resolved at this point. The containers receive GPG keys and verify signatures fine without it.